src/react/boosts/Boost.tsx

```diff
@@ -134,14 +134,14 @@ class BoostComponent extends React.Component<BoostProps, BoostState> {
   }
 
   private renderEditMode(boostData: BoostData, newsroomData: BoostNewsroomData): JSX.Element {
-    const listingUrl = "https://registry.civil.co/listing/" + boostData.channelID;
+    const listingUrl = "https://registry.civil.co/listing/" + boostData.channel.newsroom.contractAddress;
```

Comment:
is the app URL available via config or something? (not worth changing now, but maybe add a todo?

Reply:
We have a ticket for this in the backlog